docs/pages/build-reference/migrating.md

```diff
@@ -10,6 +10,18 @@ One of the goals with EAS Build is to make it as easy as possible to migrate fro
 
 EAS Build only supports SDK 41+ managed projects. You must upgrade your project to migrate to EAS Build.
 
+### Expo config `userInterfaceStyle` depends on `expo-system-ui` being installed
+
+Selecting a native appearance mode with `userInterfaceStyle` (or `android.userInterfaceStyle`) in the project `app.json` will only work on Android if `expo-system-ui` is installed in the project. This is because `expo-system-ui` includes code for locking the interface natively based on the `app.json`. This feature is only supported in **Expo SDK +43**.
```

Comment:
```suggestion
Selecting a native appearance mode with `userInterfaceStyle` (or `android.userInterfaceStyle`) in the project `app.json` will only work on Android if `expo-system-ui` is installed in the project. This is because `expo-system-ui` includes code for locking the interface natively based on the `app.json`. Run `expo install expo-system-ui` to add the library. This feature is only supported in **Expo SDK +43**.
```